api/src/main/java/org/neo4j/ogm/config/ObjectMapperFactory.java

```diff
@@ -70,6 +66,7 @@ public static JsonFactory jsonFactory() {
      * nodes- and relationshipmodules. They are used when the model is manually build. Without this mixin, they would
      * need to be called {@code setXXX}, which is totally misleading.
      */
+    @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Just for my understanding: This is set to `ignoreUnknown` because we can have either the newly added part or this in the response?